src/components/DateSelector.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div>
+    <va-date-input
+    label="due date"
+    v-model="value"
+    :format="formatDate"
+    :parse="parseFn"
+    placeholder="yyyy-mm-dd"
```

Comment:
This placeholder looks like it needs to match the formatting call below. Might be helpful to have a single const for that?